batch/rae.json

```diff
@@ -1,6 +1,6 @@
 {
-    "title": "rae",
-    "description": "rae",
+    "title": "RAE",
+    "description": "RAE",
```

Comment:
`rae` is actually lowercase "stylized". Just that we have UPPERCASE for naming of files and other name conventions

Reply:
I don't think title and description are used anywhere in the code other that the GUI, so it probably doesn't matter.